### PR TITLE
Only check toggle if it's on the enikshay env

### PR DESCRIPTION
**Pull request by esoergel**
from https://github.com/dimagi/commcare-hq/pull/18078#discussion_r142657984
@emord

### CODE REVIEW

**Review by emord — APPROVED** [review]
This seems ok, but what do you think about adding a parameter to toggles called `enabled_environments` or something similar and having a static check here: https://github.com/dimagi/commcare-hq/blob/master/corehq/toggles.py#L75

That will solve the issue everywhere and gives us an easy way to only enable certain flags on certain server environments

**Comment by esoergel** [comment]
@emord updated.  Lemme know what you think.  I'm sure there are other feature flags that will benefit from this param, but I'd like to start with this and add the others after.

File: corehq/toggles.py [hunk 1]
```diff
@@ -67,12 +67,20 @@ def __init__(self, slug, label, tag, namespaces=None, help_link=None,
         self.always_disabled = always_disabled or set()
         self.enabled_for_new_domains_after = enabled_for_new_domains_after
         self.enabled_for_new_users_after = enabled_for_new_users_after
+        # pass in a set of environments where this toggle applies
+        self.relevant_environments = relevant_environments
```
Comment (emord) [inline corehq/toggles.py]: you could initialize this as `relevant_environments or []` and get rid of the null check later

**Review by emord — APPROVED** [review]

File: corehq/toggles.py (hunk 1 above)
Comment (esoergel) [reply corehq/toggles.py]: Actually I need the null check - if it's not specified (or empty), then we need to treat it as if all environments are listed.

**Pull request merged**